modules/integration/tests/src/test/java/org/wso2/appmanager/integration/test/cases/admin/dashboard/authentication/AdminDashboardLogout.java

```diff
@@ -0,0 +1,49 @@
+package org.wso2.appmanager.integration.test.cases.admin.dashboard.authentication;
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/e263afed698e7dbd1253c0136563513fd4051eea